docs/fleet/machines/control.md

```diff
@@ -0,0 +1,109 @@
+
+For example, if you have configured a base with wheels, you can move your machine's with an arrow pad and control the base's speed by setting its power with a slider.
+If you have configured a camera component, a window in the **CONTROL** tab displays the camera output.
+
+If you use remote control in the [Viam app](https://app.viam.com) UI, all communication to the machine uses [WebRTC](https://pkg.go.dev/go.viam.com/utils@v0.0.3/rpc#hdr-Connection).
+For local communication between [parts](/build/configure/parts/#machine-parts) Viam uses gRPC or WebRTC.
+
+{{<gif webm_src="/fleet/control.webm" mp4_src="/fleet/control.mp4" alt="Using the control tab" max-width="800px">}}
+
+### Components
+
+For more detailed information on how to operate and test your resources, expand the relevant resource below:
+
+{{% expand "arm" %}}
+{{< readfile "/static/include/components/test-control/arm-control.md" >}}
+{{% /expand%}}
+
+{{% expand "base" %}}
+{{< readfile "/static/include/components/test-control/base-control.md" >}}
+{{% /expand%}}
+
+{{% expand "board" %}}
+{{< readfile "/static/include/components/board/test-board-digital-interrupts.md" >}}
+{{% /expand%}}
+
+{{% expand "camera" %}}
+{{< readfile "/static/include/components/camera-view-camera-stream.md" >}}
+{{% /expand%}}
+
+{{% expand "encoder" %}}
+{{< readfile "/static/include/components/test-control/encoder-control.md" >}}
+{{% /expand%}}
+
+{{% expand "gantry" %}}
+{{< readfile "/static/include/components/test-control/gantry-control.md" >}}
+{{% /expand%}}
+
+{{% expand "generic component" %}}
+{{< readfile "/static/include/components/test-control/generic-control.md" >}}
+{{% /expand%}}
+
+{{% expand "gripper" %}}
+{{< readfile "/static/include/components/test-control/gripper-control.md" >}}
+{{% /expand%}}
+
+{{% expand "input controller" %}}
+{{< readfile "/static/include/components/test-control/input-controller-control.md" >}}
+{{% /expand%}}
+
+{{% expand "motor" %}}
+{{< readfile "/static/include/components/test-control/motor-control.md" >}}
+{{% /expand%}}
+
+{{% expand "movement sensor (gps)" %}}
+{{< readfile "/static/include/components/test-control/movement-sensor-gps-control.md" >}}
+{{% /expand%}}
+
+{{% expand "movement sensor (imu)" %}}
+{{< readfile "/static/include/components/test-control/movement-sensor-imu-control.md" >}}
+{{% /expand%}}
+
+{{% expand "power sensor" %}}
+{{< readfile "/static/include/components/test-control/power-sensor-control.md" >}}
+{{% /expand%}}
+
+{{% expand "sensor" %}}
+{{< readfile "/static/include/components/test-control/sensor-control.md" >}}
+{{% /expand%}}
+
+{{% expand "servo" %}}
+{{< readfile "/static/include/components/test-control/servo-control.md" >}}
+{{% /expand%}}
+
+### Services
+
+The following services also provide control interfaces:
+
+- [SLAM](/mobility/slam/cartographer/#create-a-new-map): for createing a new map
+- [Motion](/mobility/motion/#test-the-motion-service): for moving a machine on a map
```

Comment:
Is this true? Shouldn't this be 
```suggestion
- [Navigation](/mobility/navigation/#control-tab-usage): for moving a machine to waypoints on a map
```
The currently-shown image on https://docs.viam.com/mobility/motion/#test-the-motion-service shows the SLAM card in the control tab. The motion service can be used for 2D motion planning on a SLAM map, but this section of the motion page might make users think that it's not for 3D motion planning too. The motion service itself doesn't have a control tab card, so this "Test the motion service" section should probably be fixed. Might make sense to fix within this PR?

Reply:
https://docs-test.viam.dev/2851/mobility/motion/#test-the-motion-service seems to indicate so, but I should add navigation